WEB-4262 - cloudcraft api (#21556)
DataDog/documentation: content/en/cloudcraft/api/_index.md

---
title: API Reference

> type: documentation

Reviewer: do we want to disable the toc for cloudcraft api docs?  (dd api docs dont have toc)

---

{{< openapi-ref-docs url="cloudcraft.json" >}}

## Client libraries

> By default, the Cloudcraft API Docs show examples in cURL. Select one of our official client library languages in each endpoint to see code examples from that library. To install each library:

Reviewer 2: ```suggestion
By default, the Cloudcraft API docs show examples in cURL. Each endpoint also includes code examples from one of the official client library languages. To install each library:
```
